client/py/synnax/timing/timing.py

```diff
@@ -45,7 +42,116 @@ def _precise_sleep(dur: TimeSpan | float | int):
         pass
 
 
-def sleep(dur: TimeSpan | float | int, precise: bool = False):
+def sleep(dur: Rate | TimeSpan | float | int, precise: bool = False):
+    """Sleep is a function that sleeps for the given duration. The duration can be
+    specified as a Rate, TimeSpan, float, or int. If the precise flag is set to True,
+    the function will use a more precise sleep implementation. It uses welford's
+    algorithm to estimate the ideal time to sleep for the given duration. This function
+    uses considerably more CPU than time.sleep, so it should only be used when high
+    precision is required.
```

Comment:
This docstring is somewhat unclear - when `precise` is `False`, it uses the same CPU time as time.sleep. I would do two things:
1. Make clear that when `dur` is a `float` or `int`, the time is in seconds.
2. Move the commentary on the precise sleep to a second paragraph to be clear that the CPU time and Welford's algorithm comment are just for precise sleep
3. Capitalize Welford.